apps/examples/wc-angular12/src/app/app.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { ButtonProps, buttonSizes, buttonVariants } from '@justeattakeaway/pie-button';
+import { ButtonProps, sizes, variants } from '@justeattakeaway/pie-button';
```

Comment:
Could this potentially get confusing if we end up importing other components with sizes and variants (for example, `pie-icon-button` has the same variables)?

Reply:
Just a small note as well – I'd recommend keeping refactoring like this in a separate PR. It makes it a bit harder to see what code is specifically to do with the scroll-locking.

Reply:
I'll definitely split in future but going to keep this PR for now if that's ok.

I don't really think itis confusing, I actually think they should be named similarly. I see it more as consistency. You'd import from different namespaces and probs end up using aliases if you had to but I don't think we should be doing that preemptively. I see it as more of an external usage concern rather than internal one.

Reply:
Regarding your first question @ashleynolan, I think that's a valid point, but in the scenario where someone needs to directly work with the different sizes and variants for multiple components they should be willing to rename the variables on import them. I don't think we need to make the names longer or more complex to cater for that.

```javascript
import { ButtonProps, sizes as buttonSizes, variants as buttonVariants } from '@justeattakeaway/pie-button';
import { IconButtonProps, sizes as iconButtonSizes, variants as iconButtonVariants } from '@justeattakeaway/pie-button';
```

Reply:
Yeah that's how I envisioned it too :)